src/engine-actions/recover-wallet.js

```diff
@@ -0,0 +1,56 @@
+const { initWallet } = require('../lnd-actions')
+
+/**
+ * Wallet recovery window number in blocks
+ * @constant
+ * @type {number}
+ * @default
+ */
+const RECOVERY_WINDOW_DEFAULT = 5000
+
+/**
+ * @param {Array<string>} seed
+ * @returns {boolean}
+ */
+async function isValidSeed (seed) {
+  if (!Array.isArray(seed)) return false
+  if (seed.length !== 24) return false
+  return true
+}
+
+/**
+ * Creates a wallet
+ *
+ * @param {string} password - wallet password, used to unlock lnd wallet
+ * @param {Array<string>} seed - 24 word cipher seed mnemonic
+ * @param {Buffer} backup - Buffer of the LND static channel backup file
+ * @param {number} recoveryWindow - number in blocks for look back
+ * @returns {Array<string>} 24 word cipher seed mnemonic
+ */
+async function recoverWallet (password, seed, backup, recoveryWindow = undefined) {
+  // The seed is required if the user is using a backup file
+  if (!seed && backup) {
```

Comment:
I might add a warning if they are using only the seed to let them know that only on-chain funds will be recovered.